src/mjlab/scripts/train.py

```diff
@@ -141,6 +142,34 @@ def run_train(task_id: str, cfg: TrainConfig) -> None:
   env.close()
 
 
+def launch_training(task_id: str, args: TrainConfig | None = None):
+  args = args or TrainConfig.from_task(task_id)
+
+  import torchrunx
+
+  # torchrunx redirects stdout to logging.
+  logging.basicConfig(level=logging.INFO)
+
+  env_vars = {"MUJOCO_GL": "egl"}
+
+  if args.gpus == "all":
+    import torch.cuda
+
+    num_gpus = torch.cuda.device_count()
+  else:
+    num_gpus = len(args.gpus)
+    env_vars["CUDA_VISIBLE_DEVICES"] = ",".join(map(str, args.gpus))
+
+  print(f"[INFO] Launching training with {num_gpus} GPUs", flush=True)
+  launcher = torchrunx.Launcher(
```

Comment:
I'm lightly biased toward only using `torchrunx` when doing >1 GPU because of:
- Startup overhead
- Simplicity / reducing indirection
    - Confidence that things like `breakpoint()`, `ipdb.set_trace()`, will just work

Reply:
+1

Reply:
Thanks @apoorvkh, will happily switch to `torchrunx` for the single-GPU case when debugging is properly supported!